conf/schemas/cloudtrail.json

```diff
@@ -22,26 +22,31 @@
   "cloudtrail:events": {
     "schema": {
       "additionalEventData": {},
+      "addendum": {},
```

Comment:
```
addendum
If an event delivery was delayed, or additional information about an existing event becomes available after the event is logged, an addendum field shows information about why the event was delayed. If information was missing from an existing event, the addendum field includes the missing information and a reason for why it was missing. Contents include the following.

reason - The reason that the event or some of its contents were missing. Values can be any of the following.

DELIVERY_DELAY – There was a delay delivering events. This could be caused by high network traffic, connectivity issues, or a CloudTrail service issue.

UPDATED_DATA – A field in the event record was missing or had an incorrect value.

SERVICE_OUTAGE – A service that logs events to CloudTrail had an outage, and couldn’t log events to CloudTrail. This is exceptionally rare.

updatedFields - The event record fields that are updated by the addendum. This is only provided if the reason is UPDATED_DATA.

originalRequestID - The original unique ID of the request. This is only provided if the reason is UPDATED_DATA.

originalEventID - The original event ID. This is only provided if the reason is UPDATED_DATA.
```